<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MusicCollectionWizard</class>
 <widget class="QWizard" name="MusicCollectionWizard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Music Collection Wizard</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWizard{
	background-color:  rgb(94, 92, 100);
}</string>
  </property>
  <widget class="QWizardPage" name="mainPage">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="0" colspan="2">
     <widget class="QLabel" name="TypeofMusicCollection">
      <property name="text">
       <string>Select type of you music collection</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="CollectionName">
      <property name="text">
       <string>Enter you collection name</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="MusicCollectionGenreInput"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="CollectionGenre">
      <property name="text">
       <string>Enter you music collection genre</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="CollectionNameInput"/>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QComboBox" name="TypeofCollectionComboBox">
      <item>
       <property name="text">
        <string>Plain Text type</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>MultiMedia type</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
